app/models/external_registries/stop_heling_client.rb

```diff
@@ -4,11 +4,13 @@ module ExternalRegistries
   class StopHelingClient
     APP_ID = ENV["STOP_HELING_APP_ID"]
     API_KEY = ENV["STOP_HELING_API_KEY"]
+    TTL_HOURS = ENV.fetch("STOP_HELING_API_TTL_HOURS", 24).to_i.hours
 
-    attr_accessor :conn, :base_url
+    attr_accessor :conn, :base_url, :request_gap_seconds
 
     def initialize(base_url: nil)
       self.base_url = base_url || ENV["STOP_HELING_BASE_URL"]
+      self.request_gap_seconds = 1
```

Comment:
I am really concerned about this, and it being in the `v3` search path.

Do you share this concern? I suppose since they're unicorn processes they will get killed eventually...

Reply:
My personal preference would be to store the api rate limited status globally (perhaps in redis?), and making it so that failures automatically retry (hopefully leveraging sidekiq's retry).

So if we're running queries in parallel they don't keep hammering the API and cause a state where things are always sleeping and never succeeding at fetching new pages.

This might be a concern that is unnecessary at this stage, it's just something that has bit me before.

Reply:
> I am really concerned about this, and it being in the v3 search path.

That sleep was added for the sake of pagination, but I don't think it needs to be there now that we're not paginating. I'll kill it.